pkg/rpc/request.go

```diff
@@ -85,32 +85,34 @@ func (r Request) WriteErrorResponse(w http.ResponseWriter, err error) {
 		ID:      r.RawID,
 	}
 
-	logFields := log.Fields{
-		"err":    jsonErr.Cause,
-		"method": r.Method,
+	logFields := []zap.Field{
+		zap.Error(jsonErr.Cause),
+		zap.String("method", r.Method),
 	}
+
 	params, err := r.Params()
 	if err == nil {
-		logFields["params"] = *params
+		logFields = append(logFields, zap.Any("params", params))
```

Comment:
Don’t use any, it produced huge allocs

Reply:
What do you suggest to use instead?

Reply:
You have more than one option to use.. int/string/other

Reply:
What do you suggest to use here?

Reply:
`for range` throw Params() and switch case by type?

Reply:
Does it make more allocs than `logrus` in similar case?

Reply:
nop, but... It encodes message as JSON and then writes to stdout, it can produce huge memallocs on an empty sit

Reply:
I don't see a necessity to do it right now. We will do it in another PR.

Reply:
OK, it's a microoptimization, we may not bother for now if it simplifies transition from logrus.